### ISSUE TITLE
NTR: Decreased expression of GPI-anchored proteins on the cell surface

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4659]
**Preferred term label:**
Decreased expression of GPI-anchored proteins on the cell surface

**Synonyms**
N/A

**Definition (free text, please give PubMed ID)**
A decrease in the protein expression fo GPI-anchor proteins, such as CD55 and CD59, at the cell surface, which suggests a defect in GPI-anchor biosynthesis.
Source: PMID:26293662

**Parent term (use hpo.jax.org/app)**
HP_0011017 'Abnormal cellular physiology'
_(is this the best parent?)_

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/616809?search=616809&highlight=616809
https://www.omim.org/entry/618010?search=%20618010&highlight=618010#2
https://www.omim.org/entry/614080?search=614080&highlight=614080
(Note - I am reviewing the annotations for diseases listed in the GPI paper)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

@pnrobinson I can add this if it looks okay
